CAVEATS.md

```diff
@@ -0,0 +1,52 @@
+# Platform specific notes
+
+## Windows
+
+### Spectest requirements
+
+You need recent version of either Strawberry Perl or ActiveState Perl.
```

Comment:
How effective is it to move some old pieces of text from one location to another! We need to do it time after time to correct otherwise overlooked errors! :)